<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>single_page_ui</class>
 <widget class="QMainWindow" name="single_page_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smartify File Deleter</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0729a3;
color: #ffffff;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>481</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>30</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SMARTIFY FILE DELETER</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>351</width>
      <height>311</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;
</string>
    </property>
    <property name="title">
     <string>CONDITIONS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QCheckBox" name="is_recursion_needed">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>250</y>
       <width>271</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Recurse In to sub directories</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>281</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>File name (enter the starting letters)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="extensions_to_be_deleted">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>jpg, exe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="file_starting_name">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>abc</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;b&gt;*&lt;/b&gt;set conditions for the deletion,leave it empty if you dont &lt;br/&gt; want  to use a filter&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Extensions (separated by comma) &lt;br/&gt;for example .jpg, .exe&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff;

</string>
    </property>
    <property name="title">
     <string>SELECT FOLDER</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="chooseFolderButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>80</y>
       <width>21</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>..</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>47</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Folder Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="choosedFolder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>271</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #000000;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="affectedFilesList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>301</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="affectedFileStatusArea">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Affected FIles</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="confirm_delete">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>420</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #0729a3;
color: #ffffff;
padding: 10px;</string>
    </property>
    <property name="text">
     <string>CONFIRM DELETE</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
